mint-auditor/src/http_api/mod.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! HTTP server for mint auditor
+
+/// Routes for the HTTP server
+mod routes;
+
+/// Service for handling HTTP requests
+mod service;
+
+/// Request and response types
+mod api_types;
+
+/// Start the HTTP server
+pub mod launch_rocket;
```

Comment:
I think the `start_http_server` method should just be brought into this file, but if not this should be changed to `mod launch_rocket` and `pub use launch_rocket::start_http_server`. (and the convention in the codebase is to group all `mod`s together and then all `pub use` together.